features-json/push-api.json

```diff
@@ -308,7 +308,7 @@
       "15.4":"n #3",
       "15.5":"n #3",
       "15.6":"n #3",
-      "16.0":"a #6",
+      "16.0":"n #3",
       "16.1":"a #6",
```

Comment:
Here, I would change 16.1 to `a #3 #6`, because Safari will continue to support Safari Push Notifications for the foreseeable future. We are not dropping support. And I wouldn't want web developers to get worried, if the `#3` were to disappear.

Reply:
Thank you, have adjusted including note 3, hope it's good this way!